packages/typespec-ts/package.json

```diff
@@ -18,13 +18,14 @@
     "test-next": "vitest run ./test-next",
     "test-next:coverage": "vitest run ./test-next --coverage",
     "clean": "rimraf ./dist ./typespec-output",
-    "build": "tsc -p .",
+    "build": "tsc -p . && npm run copy-static-helpers",
```

Comment:
The command may be not working in windows environement. This error can't be catched in our ci. I think we should at least enable one windows environement.

```
==[ BLOCKED: 1 operation ]=====================================================

These operations were blocked by dependencies that failed:
  @msinternal/typespec-test

==[ FAILURE: 1 operation ]=====================================================

--[ FAILURE: @azure-tools/typespec-ts ]----------------------[ 5.24 seconds ]--

Error copying files from C:\C:\Users\marygao\project\autorest.typescript\packages\typespec-ts\src\modular\static-helpers to C:\C:\Users\marygao\project\autorest.typescript\packages\typespec-ts\dist\src\modular\static-helpers: Error: ENOENT: no such file or directory, scandir 'C:\C:\Users\marygao\project\autorest.typescript\packages\typespec-ts\src\modular\static-helpers'        
    at async readdir (node:internal/fs/promises:949:18)
    at async copyFiles (file:///C:/Users/marygao/project/autorest.typescript/packages/typespec-ts/eng/scripts/copy-static-helpers.js:8:21)
    at async file:///C:/Users/marygao/project/autorest.typescript/packages/typespec-ts/eng/scripts/copy-static-helpers.js:35:3 {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'scandir',
  path: 'C:\\C:\\Users\\marygao\\project\\autorest.typescript\\packages\\typespec-ts\\src\\modular\\static-helpers'
}


Operations failed.

rush build (11.63 seconds)
```

Reply:
Thanks for catching this I have filed an issue #2723 to track CI validation in Windows. Probably we don't want to run everything on windows but maybe one smoke-test can help us give good coverage for the platform differences